PowerUp/app/src/main/res/layout-land/activity_scenario_over.xml

```diff
@@ -18,6 +18,15 @@
         android:layout_marginLeft="@dimen/replay_marginLeft"
         android:alpha="0"/>
 
+    <TextView
+        android:id="@+id/scenarioName"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/karma_star"
```

Comment:
Error:(25, 31) No resource found that matches the given name (at 'layout_below' with value '@id/karma_star').

Reply:
I have fixed the error. :)